smoke_tests/fixtures/requests.go

```diff
@@ -7,6 +7,16 @@ import (
 	"github.com/Admiral-Piett/goaws/app/models"
 )
 
+var CreateQueuesRequestBodyXML = struct {
+	Action    string `xml:"Action"`
+	Version   string `xml:"Version"`
+	QueueName string `xml:"QueueName"`
+}{
+	Action:    "CreateQueue",
+	Version:   "2012-11-05",
+	QueueName: "new-queue-1",
+}
```

Comment:
Do we still need this?  I thought I took this out.  I think it's ok to use the JSON sdk for creating queues when testing other things?  I only particularly care about making sure the XML works in the dedicated smoke tests.

Reply:
I think it's okay to delete it. At least in the SendMessage test, CreateQueue using the SDK works fine, including setting MaximumMessageSize!
I believe that testing with XML has been confirmed in existing tests even before we started this work.

Reply:
I realized `sqs_create_queue_test.go` is the only test that use XML (form protocol) since `create_queue_test.go` only has UT that is separated from any protocol.

If future goaws also supports XML, I think we should keep it for XML testing. However, for other tests such as SendMessage, it is sufficient to be able to create a queue with the SDK, so I think this XML definition can be moved into `sqs_create_queue_test.go` as private (starting with a lowercase letter).

Reply:
Finally, I haven't used it. I removed it 👍